packages/core/src/app/payment/paymentMethod/WorldpayCreditCardPaymentMethod.tsx

```diff
@@ -58,8 +58,8 @@ const WorldpayCreditCardPaymentMethod: FunctionComponent<
     }, [initializePayment]);
 
     const appendPaymentPageContent = useCallback(() => {
-        if (threeDSecureVerification) {
-            ref.current.paymentPageContentRef.current?.appendChild(threeDSecureVerification);
+        if (threeDSecureVerification && ref.current.paymentPageContentRef.current) {
+            ref.current.paymentPageContentRef.current.appendChild(threeDSecureVerification);
```

Comment:
Can I please ask what's the difference in the two statements?

Reply:
Code coverage:
![image](https://user-images.githubusercontent.com/4907027/189847886-261e6e36-4c9b-4152-96fc-6302710148ce.png)